docs/source/upgrade-guide/index.md

```diff
@@ -2316,7 +2322,7 @@ Modify the parent element of your block making these changes:
 - Add a proper role for it
 
 ```jsx
-  role="presentation"
+  role='presentation';
 ```
```

Comment:
I am OK with the MyST and English syntax and grammar changes.

I will leave the JSX linting to others, as I am not familiar with the automatic linting tool's results. But this does raise an interesting question: How should we format and lint code snippets in documentation, and check that they conform to our guidelines? Is there such a tool that can check this? I use PyCharm which lacks a decent MyST syntax checker, and I haven't tried VS Code's extension for MyST.

Reply:
In Volto it is quite easy because prettier handles that, so it has the same configuration as the codebase. That's what happened here and I am actually surprised it did not happen earlier. I don't know about other Plone repos though.

Reply:
I am surprised as well. Perhaps prettier only checks files that are edited? And why does GitHub Actions run prettier, but not on documentation code snippets?